source/common/res/features/popup-calculator/budget/budget_calc.css

```diff
@@ -1,8 +1,15 @@
+#toolkitPopupCalcInput  {
+    margin-top: .3em;
+    margin-bottom: .3em;
+    width: 176px;
+		text-align: right;
+}
+
 .budget-table-cell-activity {
   width: 17% !important;
 }
 
-.toolkit-popup-calc-button {
+.ytoolkit-popup-calc-button {
```

Comment:
I think this is probably a typo, as elsewhere this is referred to with the selector `toolkit-popup-calc-button`. We should either make this CSS rule apply, or remove it if it's not needed.